src/libsyntax/ext/tt/macro_rules.rs

```diff
@@ -117,12 +117,18 @@ impl<'a> MacResult for ParserAnyMacro<'a> {
 
     fn make_stmts(self: Box<ParserAnyMacro<'a>>)
                  -> Option<SmallVector<ast::Stmt>> {
+        let parse_stmt = |parser: &mut Parser<'a>| -> ::parse::PResult<'a, _> {
```

Comment:
Could you comment on what this closure does (i.e., why you don't just use parser.parse_stmt)?

Could it be a function rather than a closure?